ccloud/beginner-cloud/README.md

```diff
@@ -29,6 +30,16 @@ If you choose to run it against your Confluent Cloud cluster, be aware that it:
 - is for demo purposes only
 - should be used only on a non-production cluster
 
+## Pre-requisites
+
+* Access to a Confluent Cloud cluster
+* Local install of the new [Confluent Cloud CLI](https://docs.confluent.io/current/cloud/cli/install.html#ccloud-install-cli?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud) v0.192.0 or above
```

Comment:
casual and _very_ minor observation... we have a mix of terms when discussing version numbers.  "or higher", "or above", "or later".  Maybe we could settle on a term for this.  Personally, I prefer "or later", but am open to any of them.  I figure if we agree on a term we can converge on it being standard over time.

Reply:
Ack.  Switch to `later`